crates/starknet-devnet-cli/tests/get_transaction_by_hash.rs

```diff
@@ -23,8 +23,6 @@ mod get_transaction_by_hash_integration_tests {
     use crate::common::utils::{get_deployable_account_signer, resolve_path};
 
     #[tokio::test]
-    #[ignore = "Starknet-rs does not support estimate_fee with simulation_flags, but providing a \
```

Comment:
Doesn't Cargo.toml need to be updated to get a new starknet-rs version which supports RPC 0.6?

Reply:
the current version, supports spec 0.6

Reply:
Oh really, addressed in #294